.github/workflows/build.yml

```diff
@@ -1,20 +1,37 @@
-name: build
-on:
-  workflow_call:
-    inputs:
-      project:
-        required: true
-        type: string
-      vars:
-        type: string
+name: Building
```

Comment:
Do these changes have anything to do with G530M porting?

Reply:
Well, I wanted to modify my repository so that the artifacts can be downloaded, I didn't know that it would be reflected, how do I prevent these changes from occurring?

Reply:
For online helps, please feel free to come back to [#msm8916:postmarketos.org](https://app.element.io/#/room/#msm8916:postmarketos.org) .

Maybe you can push it to another branch, with `git switch -c <new_branch>` and `git push <your_repo> [new_branch]`. Uppercase `-C` to overwrite, if necessary.

For this PR, you can also try to create another branch `fortunalteub` with:
1. `git checkout origin/master` to checkout to `origin/master`
2. `git switch -c fortunalteub` to switch and create branch `fortunalteub`
3. `git cherry-pick ddea3ace6cf69768627ea19506398254d52ab9d6 b6f59726c60a9bca5365d26ac1b309d0b39b1d28` to cherry-pick 2 relevant commits
4. `git rebase -i origin/master` to rebase
5. In the editor:
```
pick ddea3ace6cf69768627ea19506398254d52ab9d6  ...
# fixup with previous commit
fixup b6f59726c60a9bca5365d26ac1b309d0b39b1d28 ...
# execute git commit --amend -m <commit_message>
exec git commit --amend -m "dts: Add Samsung Galaxy Grand Prime (SM-G530Y)"
```
6. `git push <your_repo>`, and open another clean PR.
    - or `git push <your_repo> fortunalteub:master` to replace your current branch `master` in this PR